src/test/regress/expected/am_truncate_0.out

```diff
@@ -1,256 +0,0 @@
---
```

Comment:
I guess this test output was specific to pg <=10 so we're removing it ?

Reply:
<= 11, anyway.